ibm/config.go

```diff
@@ -689,6 +689,20 @@ func (sess clientSession) keyProtectAPI() (*kp.Client, error) {
 }
 
 func (sess clientSession) keyManagementAPI() (*kp.Client, error) {
+
+	if sess.kmsErr == nil {
+		clientConfig := &kp.ClientConfig{
+			BaseURL: envFallBack([]string{"IBMCLOUD_KP_API_ENDPOINT"}, sess.kmsAPI.Config.BaseURL),
```

Comment:
I think here we need to check two ways...
if APIkey then create session with APIkey else use the bearer token